external-crates/move/crates/language-benchmarks/tests/arith.move

```diff
@@ -2,6 +2,10 @@ module 0x1::bench {
     const COUNT: u64 = 10_000u64;
     const MAX_U64: u64 = 18446744073709551615;
 
+    fun check(check: bool, code: u64) {
```

Comment:
For this I think changing the `check` call to an `assert!` would probably be better (it will remove the function call to `check` which will make this more closely track to just measuring arithmetic overhead IMO)